ui/src/app/pages/access/access-renewal.spec.tsx

```diff
@@ -97,10 +97,10 @@ describe('Access Renewal Page', () => {
   };
 
   beforeEach(() => {
-    const profileApi = new ProfileApiStub();
+    const profileApiImpl = new ProfileApiStub();
 
-    registerApiClient(ProfileApi, profileApi);
-    mockUpdateProfile = jest.spyOn(profileApi, 'updateProfile');
+    registerApiClient(ProfileApi, profileApiImpl);
+    mockUpdateProfile = jest.spyOn(profileApi(), 'updateProfile');
```

Comment:
this is not the same `profileApi`

Reply:
FYI: It is the same value in the case, due to the `register` call on 102. I think your way of writing it is slightly clearer though.